pom.xml

```
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.gravitee</groupId>
+        <artifactId>gravitee-parent</artifactId>
+        <version>20.2</version>
```

We have released a 20.3 version of the `gravitee-parent` which allow to be more flexible regarding maven enforcer plugin. See https://github.com/gravitee-io/gravitee-parent/commit/5c0cd925fe14f6ff7f769558593cfd1b987d94cf
```suggestion
        <version>20.3</version>
```

---

Do we have doc somewhere to know exactly what do we have to do when we are creating a new policy instead of relying only on reviews ?
Also, the maven archetype is still using 20.2 : https://github.com/gravitee-io/gravitee-policy-maven-archetype/blob/master/src/main/resources/archetype-resources/pom.xml#L33

---

No docs for the moment, and the 20.3 is very new. I will open a PR on the archetype.

---

done